### ISSUE TITLE
[feature request] support tiff

### ISSUE THREAD

**Issue by bung87** [DavideGalilei/nimtesseract#1]
ref: https://github.com/tesseract-ocr/tessdoc/blob/main/InputFormats.md

**Comment by DavideGalilei** [comment]
I am currently using the pixie library to read images. My guess is that it doesn't work yet for tiff images, but I didn't try it out yet. If that us the case, please consider opening an issue on https://github.com/treeform/pixie, asking for TIFF images support

**Comment by DavideGalilei** [comment]
> is there a way passing image with meta info and raw bytes to tesseract

Yes, it is possible, however you would need to use the raw tesseract api. I provided a minimal example on how to do that here: https://github.com/DavideGalilei/nimtesseract/blob/master/examples/advanced.nim#L30
